presto-main/src/test/java/com/facebook/presto/execution/TestNodeScheduler.java

```diff
@@ -577,6 +585,57 @@ public void testAffinityAssignment()
         assertEquals(splitPlacementResult.getAssignments().keySet().size(), 3);
     }
 
+    @Test
+    public void testAffinityAssignmentWithConsistentHashing()
```

Comment:
Can we also add test for the virtual node? right now in this test, it would also generate one place in the hashing ring for each node

Reply:
It's a bit harder to reason when more nodes. I'll try.